js/tinymce/plugins/charmap/plugin.js

```
@@ -0,0 +1,1658 @@
+    var global$1 = tinymce.util.Tools.resolve('tinymce.PluginManager');
+
+    const fireInsertCustomChar = (editor, chr) => {
+      return editor.dispatch('insertCustomChar', { chr });
+    };
+
+    const insertChar = (editor, chr) => {
+      const evtChr = fireInsertCustomChar(editor, chr).chr;
+      editor.execCommand('mceInsertContent', false, evtChr);
+    };
+
+    const hasProto = (v, constructor, predicate) => {
```

Biome (1.9.4)

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)